TypeScript.YAML-tmLanguage

```diff
@@ -273,8 +273,10 @@ repository:
     end: '\)'
     patterns:
     - include: '#comment'
+    - include: '#parameter-name'
```

Comment:
Yeah sure. I have also written a test file the listed all the common cases for scope detection which I can show you tomorrow.